tests/app/test_registration.py

```diff
@@ -252,7 +252,7 @@ def test_create_account_get_page_with_inactive_enrolment_code(self, mock_object)
                                 follow_redirects=True)
 
         self.assertEqual(response.status_code, 500)
-        self.assertTrue('Server error'.encode() in response.data)
+        self.assertTrue('An error has occurred'.encode() in response.data)
```

Comment:
At this stage, you've referred to `'An error has occurred'.encode()` so often that I think declaring a variable of it earlier in test setup would be worthwhile to make for ease of changing it.

Reply:
Added to config, just a method that returns a string now but a string file would be better in future